scout/constants/variants_export.py

```diff
@@ -19,6 +19,9 @@
     "AD Alternative",
 ]
 
+# Used only if app is connected to chanjo report
```

Comment:
I'm not sure that the comment is warranted? If you feel it is, perhaps describe rather that it contains human readable descriptions of computed coverage stats for the MT reports?

Reply:
But good that it is a constant!